easy_phi/tests/run_tests.py

```diff
@@ -1,12 +1,46 @@
 # -*- coding: utf-8 -*-
-
+import tornado.httpserver
+import tornado.httpclient
+import tornado.ioloop
+import tornado.web
 import unittest
+import json
+from easy_phi import app as handlers
 
 class API_test(unittest.TestCase):
 
     def test_t(self):
         self.assertTrue(True)
 
+class TestTornadoWeb(unittest.TestCase):
+    http_server = None
+    response = None
+
+    def setUp(self):
+        application = tornado.web.Application([
+                (r'/api/v1/', handlers.PlatformInfoHandler),
```

Comment:
Let me quote the discussion under this link:
 
> I'm not sure what Tornado looked like in 2009, but nowadays we have the `testing` module to provide TestCase superclasses for unit testing. Look in tornado/test/web_test.py for examples.


Actually, reuse of tornado.testing.AsyncHTTPTestCase is a great idea that will make testing much easier.